dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/MainController.java

```diff
@@ -3,24 +3,19 @@
 import java.util.Date;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 
 @Controller
 public class MainController {
 
-  @RequestMapping(
-      value = {"", "/"},
-      method = RequestMethod.GET)
+  @GetMapping(value = {"", "/"})
   public String printWelcome(Model model) {
     model.addAttribute("time", new Date());
     return "main";
   }
 
-  @RequestMapping(
-      value = {"/login"},
-      method = RequestMethod.GET)
+  @GetMapping(value = {"/login"})
```

Comment:
Wenn nur ein Wert im `value` ist und sonst kein Parameter gesetzt ist, geht auch

```suggestion
  @GetMapping("/login")
```